perun-db/postgres.sql

```diff
@@ -509,33 +500,12 @@ create table specific_user_users (
   constraint specifu_u_status_chk check (status in ('0','1'))
 );
 
--- EXEC_SERVICES - controlling data for service
-create table exec_services (
-	id integer not null,
-	service_id integer not null,    --identifier of service (services.id)
-	default_delay integer not null, --delay to repeating of service in case of error
-	enabled char(1) not null,       --service is active/inactive
-	default_recurrence integer not null,  --number of repeating in case of error
-	script varchar(256) not null,   --name of executable service script
-	type varchar(10) not null,      --part of service (SEND/GENERATE)
-	created_at timestamp default statement_timestamp() not null,
-	created_by varchar(1300) default user not null,
-	modified_at timestamp default statement_timestamp() not null,
-	modified_by varchar(1300) default user not null,
-	status char(1) default '0' not null,
-	created_by_uid integer,
-	modified_by_uid integer,
-	constraint exsrv_pk primary key(id),
-  constraint exsrv_srv_fk foreign key (service_id) references services(id),
-  constraint exsrv_type_chk check (type in ('SEND','GENERATE'))
-);
-
 -- SERVICE_DENIALS - services excluded from ussage
 create table service_denials (
 	id integer not null,
-	exec_service_id integer not null,  --identifier of service (exec_services.id)
-	facility_id integer,               --identifier of facility (facilities.id)
-	destination_id integer,            --identifier of destination (destinations.id) if service is not excluded on whole facility
+	service_id integer not null,       --identifier of service (services.id)
+	facility_id integer,               --identifier of service (facilities.id)
```

Comment:
Column "facility_id" should not have comment "identifier of service (services.id)"